go/vt/vtgr/db/mysql.go

```diff
@@ -0,0 +1,535 @@
+/*
+Copyright 2021 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package db
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"strconv"
+	"strings"
+
+	gouuid "github.com/pborman/uuid"
```

Comment:
We are using `github.com/google/uuid` elsewhere in the code. Any reason you can't use that?

Reply:
https://github.com/vitessio/vitess/blob/main/go/vt/workflow/manager.go#L28 is using github.com/pborman/uuid as well :-)

I will change to google/uuid since they generate same results for our purpose: https://play.golang.org/p/dAEaWBqxlgT